### PR TITLE
fixed slider and created footer

### PR DESCRIPTION
**Pull request by Ksenaaa**

### CODE REVIEW

File: src/components/Catalog/Catalog.tsx [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+import { routes } from 'utils/constants/routes'
+
+import styles from './Catalog.module.scss'
+
+type Props = {
+  onClick?: () => void;
+  styleCatalog: string
+}
+
+type menuNameType = {
```
Comment (vell28) [inline src/components/Catalog/Catalog.tsx]: all types start with a capital letter

File: src/components/Slider/Slider.module.scss [hunk 2]
```diff
@@ -7,11 +7,10 @@
   
   .containerSlider {
     height: 100%;
-    overflow-x: scroll;
-    overflow: hidden;
     display: flex;
     flex-direction: column;
     flex-wrap: wrap;
+    transition: ease all 0.5s;
```
Comment (vell28) [inline src/components/Slider/Slider.module.scss]: time to variable

File: src/components/Catalog/Catalog.tsx (hunk 1 above)
Comment (Ksenaaa) [reply src/components/Catalog/Catalog.tsx]: fixed

**Review by vell28 — APPROVED** [review]

File: src/components/Slider/Slider.module.scss (hunk 2 above)
Comment (Ksenaaa) [reply src/components/Slider/Slider.module.scss]: fixed

**Pull request merged**